config.rb

```diff
@@ -179,4 +179,7 @@ def self.test?
 
```

Comment:
typo in the commit message; `ibi-cloud`.

Reply:
That was intentional (FWIW ibi is used for Turkey, the animal in a local dialect)